### PR TITLE
CQW-159 Refactor Menu

### PR DESCRIPTION
**Pull request by pjar**
## TO DO:
* [x] Remove hammer.js
* [x] Make menu look the same on all devices
* [x] Refactor - remove overlapping styles, use mobile-first approach

---

## Review:

* [ ] @beatadelura 
* [x] @MagdaMalinowska 
* [x] @kruszczynski 
* [x] @rpietraszko 
* [ ] @irene50288

### CODE REVIEW

File: source/javascripts/application.js.coffee [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #= require_tree .
 #= require 'lib/jquery.min'
-#= require 'lib/hammer.min'
```
Comment (rpietraszko) [inline source/javascripts/application.js.coffee]: @pjar - how swipe is going to be handled now?

Comment (pjar) [inline source/javascripts/application.js.coffee]: It won't :) It was decided to get rid of that, as it didn't make a lot of sense. Showing it by swipe was not obvious for new users and hiding it was triggering going back to previous page.

Comment (rpietraszko) [inline source/javascripts/application.js.coffee]: oh I see, didn't know that :)

**Comment by rpietraszko** [comment]
I found a bug (or a feature?)

When you hover on close button it causes whole content goes up by few pixels:
http://screencast.com/t/Ild8nh2LN5

**Comment by beatadelura** [comment]
768px - 926px breaking text

zrzut ekranu 2016-02-26 o 15 35 49

**Comment by beatadelura** [comment]
Menu is not working on Safari.

File: source/svg/main_menu/main_menu_close_button.html.slim [hunk 2]
```diff
@@ -1,4 +1,4 @@
 svg(x='0px' y='0px' width='27.9px' height='27.9px' viewBox='0 0 27.9 27.9' style='enable-background:new 0 0 27.9 27.9;' xml:space='preserve')
-  g
-    circle(class='fill-white' cx='25.9' cy='25.9' r='2')
-    path(class='fill-white' d='M22.4,19.6L16.8,14L27.4,3.4c0.8-0.8,0.8-2,0-2.8c-0.8-0.8-2-0.8-2.8,0L14,11.1L3.4,0.6c-0.8-0.8-2-0.8-2.8,0c-0.8,0.8-0.8,2,0,2.8L11.1,14L0.6,24.5c-0.8,0.8-0.8,2,0,2.8c0.8,0.8,2,0.8,2.8,0L14,16.8l5.6,5.6c0.8,0.8,2,0.8,2.8,0C23.2,21.6,23.2,20.3,22.4,19.6z')
+  g.is-current-color
```
Comment (MagdaMalinowska) [inline source/svg/main_menu/main_menu_close_button.html.slim]: +1

File: source/stylesheets/modules/_m-main-menu-close-button.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
+// scss-lint:disable SelectorFormat
+// scss-lint:disable SelectorDepth
+// scss-lint:disable NestingDepth
```
Comment (kruszczynski) [inline source/stylesheets/modules/_m-main-menu-close-button.scss]: That's indicates a deeper problem :)

**Pull request merged**